config/Zed/Oms/UnzerSofort01.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<statemachine
+    xmlns="spryker:oms-01"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="spryker:oms-01 http://static.spryker.com/oms-01.xsd">
+
+    <process name="UnzerSofort01" main="true">
+
+        <states>
+            <state name="new" reserved="true"/>
+            <state name="payment pending" reserved="true"/>
+            <state name="payment completed" reserved="true"/>
+            <state name="charge failed"/>
+            <state name="refunded"/>
+            <state name="payment chargeback"/>
+            <state name="shipped"/>
+            <state name="closed"/>
+        </states>
+
+        <events>
+            <event name="charge" manual="true" command="Unzer/Charge"/>
+            <event name="refund" manual="true" command="Unzer/Refund"/>
+            <event name="ship" manual="true"/>
+            <event name="close" manual="false" timeout="14 days"/>
+        </events>
+
+        <transitions>
+
+            <transition happy="true">
+                <source>new</source>
+                <target>payment pending</target>
+            </transition>
+
+            <transition condition="Unzer/IsPaymentCompleted"  happy="true">
+                <source>payment pending</source>
+                <target>payment completed</target>
+            </transition>
+
+            <transition condition="Unzer/IsChargeFailed"  happy="true">
+                <source>payment pending</source>
+                <target>charge failed</target>
+            </transition>
+
+            <transition condition="Unzer/IsPaymentChargeback" happy="true">
+                <source>payment completed</source>
+                <target>payment chargeback</target>
+            </transition>
+
+            <transition happy="true">
+                <source>payment completed</source>
+                <target>refunded</target>
+                <event>refund</event>
+            </transition>
+
+            <transition happy="true">
+                <source>payment completed</source>
+                <target>shipped</target>
+                <event>ship</event>
+            </transition>
+
+            <transition happy="true">
+                <source>shipped</source>
+                <target>closed</target>
+                <event>close</event>
+            </transition>
+        </transitions>
+    </process>
+</statemachine>
```

Comment:
Same question here for happy attribute as for `config/Zed/Oms/UnzerBankTransfer01.xml` above